build/run

```diff
@@ -24,6 +24,16 @@ if [ ! -z $DOCKER_HOST ]; then
     return 1
 fi
 
+DOCKER_CLIENT_API=1.24   # keep in sync with container's version of docker
+set +e
+DOCKER_SERVER_API=$(docker version --format '{{.Client.APIVersion}}' 2>/dev/null)
```

Comment:
Ups, correct. I'll fix. Though they are unlikely to be different on the same OS (unless docker was updated without restarting the daemon).